lab7/download_files.py

```diff
@@ -0,0 +1,35 @@
+from urllib import request
+
+def download_url_file(url0):
+    file_open = request.urlopen(url0)
+    chunk_size = 1024 * 10
+    while True:
+        file_info = str(file_open.read(chunk_size).decode('utf-8')).split("\n")
+        if not file_info:
+            break
+        return file_info
```

Comment:
you only return the last chunk of the file, what is gonna happen to the reset of the chunks?